diesel/src/pg/types/mod.rs

```diff
@@ -297,11 +297,11 @@ pub mod sql_types {
     /// #     )").unwrap();
     /// let new_item = NewItem {
     ///     name: "Shiny Thing".into(),
-    ///     price: Cents(123456),
+    ///     price: Cents(123_456),
```

Comment:
Also, this line and the other line in this file were not caught by `bin/test`, but I changed them because I assume they should have been linted the same way as the rest of the code, and in the future they might be.